cumulusci/tasks/release_notes/generator.py

```diff
@@ -186,8 +211,10 @@ def _update_release_content(self, release, content):
                 if parser_content and not parser.replaced:
                     new_body.append(parser_content + "\r\n")
 
-            content = u"\r\n".join(new_body)
-
+        # add empty PRs section
+        if self.include_empty_pull_requests:
+            new_body.extend(self._render_empty_pr_section())
```

Comment:
If the task is run twice we'll end up with 2 copies of this list in the release notes. That's probably not a huge problem since typically this task is only run once by automation. But worth being aware of that difference from how the other parser sections are handled.